sound/soc/sof/ipc4-loader.c

```diff
@@ -423,6 +423,62 @@ int sof_ipc4_reload_fw_libraries(struct snd_sof_dev *sdev)
 	return ret;
 }
 
+/**
+ * sof_ipc4_update_basecfg_from_manifest - Update the base config from manifest
+ * @sdev: SOF device
+ * @fw_module: pointer struct sof_ipc4_fw_module to parse
+ * @basecfg: Pointer to the base_config to update
+ *
+ * Note: currently only used to update the CPC value if it is not set from
+ * topology already.
+ */
+void sof_ipc4_update_basecfg_from_manifest(struct snd_sof_dev *sdev,
+					   struct sof_ipc4_fw_module *fw_module,
+					   struct sof_ipc4_base_module_cfg *basecfg)
+{
+	const struct sof_man4_module_config *fm_config;
+	u32 cpc_pick = 0;
+	u32 max_cpc = 0;
+	int i;
+
+	/* The CPC is set from topology, ignoring manifest */
+	if (basecfg->cpc)
+		return;
+
+	if (!fw_module || !fw_module->fm_config)
+		return;
```

Comment:
what happens in this case? Why don't we use the ibs*obs/1000 in this case?

Reply:
With the new version we are going to use that at the end.